spec/writer_spec.rb

```diff
@@ -2,6 +2,21 @@
 
 describe M3u8::Writer do
   it 'should render master playlist' do
+    options = { playlist: 'playlist_url', bitrate: 6400,
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.